app/views/contentful/notes/_note.html.slim

```diff
@@ -0,0 +1,15 @@
+div id="#{dom_id note}"
+  h3.govuk-heading-m= note.logged_at
+
+  p
+    <strong>Page:</strong>
```

Comment:
Is SLIM not happy with that tag?

Reply:
Don't know - what's the SLIM way of doing that?  This is just a copy of `_note` first introduced 8 months ago, to be in the right folder for the new notes controller.

Reply:
`strong Page:`

Reply:
Yea, still not completely sold on SLIM - that's just looks wrong.